### PR TITLE
chore: add some info about how swaps work

### PR DESCRIPTION
**Pull request by silasbw**

### CODE REVIEW

**Comment by silasbw** [comment]
Screenshot_20230221-152844

File: branding/celo/src/brandingConfig.ts [hunk 1]
```diff
@@ -15,3 +15,4 @@ export const CELO_SUPPORT_EMAIL_ADDRESS = 'support@celo.org'
 export const SUPERCHARGE_LEARN_MORE = 'https://valoraapp.com/support/supercharge-learn-more'
 export const INVITE_REWARDS_LEARN_MORE = 'https://valoraapp.com/support/invite-rewards-learn-more'
 export const DAPPS_LEARN_MORE = 'https://valoraapp.com/support/dapps-learn-more'
+export const SWAP_LEARN_MORE = 'https://valoraapp.com/support/swap-learn-more'
```
Comment (MuckT) [inline branding/celo/src/brandingConfig.ts:18]: [valora-app-branding](https://github.com/valora-inc/valora-app-branding) should be updated too as in https://github.com/valora-inc/valora-app-branding/commit/74d31b4513a868793c453a0d9c92e34314730a24.

File: src/swap/SwapScreen.tsx [hunk 2]
```diff
@@ -307,6 +322,19 @@ const styles = StyleSheet.create({
   mutedHeader: {
     color: colors.gray3,
   },
+  disclaimerWrapper: {
```
Comment (MuckT) [inline src/swap/SwapScreen.tsx:325]: Wrap should be added here for large fonts and long localizations.

File: branding/celo/src/brandingConfig.ts (hunk 1 above)
Comment (jh2oman) [reply branding/celo/src/brandingConfig.ts:18]: added https://github.com/valora-inc/valora-app-branding/pull/16

**Comment by MuckT** [comment]
One more change is we have to update the branding valora_branding_sha in `scripts/sync_branding.sh` once the branding PR is merged.

**Review by MuckT — APPROVED** [review]

**Pull request merged**